src/components/TextareaComposer/TextareaComposer.tsx

```diff
@@ -143,8 +143,8 @@ export const TextareaComposer = ({
         return;
       }
 
-      if (event.key === 'Enter') {
-        // allow next line only on Shift + Enter. Enter is reserved for submission.
+      if (event.key === 'Enter' && !event.shiftKey) {
+        // Allow next line only on Shift + Enter. Enter is reserved for submission.
         event.preventDefault();
       }
```

Comment:
This is what `shouldSubmit` is for - this way we hard-code the behavior and if users decide to have shift+enter as an actual submit, they cannot really change it.

Reply:
Ok, I will move should submit higher and if the combination is not specified as shift + Enter then this can be executed

Reply:
Update: I had to just move the preventDefault call inside the condition that checks for submissions being open. When a submission is selected we want to prevent the default behavior of the textarea to add a new line.